src/main/java/List.java

```diff
@@ -0,0 +1,84 @@
+public class List {
+    
+    private Task[] tasks;
+    private int size;
+
+    public List() {
+        tasks = new Task[100];
+        size = 0;
+    }
+
+    public void addTodo(String name) {
+        tasks[size] = new Todo(name);
+        size++;
+        printAfterAddingTask(tasks[size - 1]);
+    }
+
+    public void addDeadline(String name, String by) {
+        tasks[size] = new Deadline(name, by);
+        size++;
+        printAfterAddingTask(tasks[size - 1]);
+    }
+
+    public void addEvent(String name, String from, String to) {
```

Comment:
i think it would be nice to add more comments to explain the methods and classes for better understanding